docs/docs/querying.md

```diff
@@ -123,6 +123,7 @@ $gte: 6,               // >= 6
 $lt: 10,               // < 10
 $lte: 10,              // <= 10
 $ne: 20,               // != 20
+$eq: 3,                // = 3
```

Comment:
This might lead som users to believe that `$eq` is required for regular equals. Perhaps show a couple of examples from https://github.com/sequelize/sequelize/blob/cd2e38e48e3054ba7e9661181f74d4ffdae65fe4/test/unit/sql/where.test.js